test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -146,4 +150,22 @@ Describe "Basic Send-MailMessage tests" -Tags CI {
         $mail.Body.Count | Should -BeExactly 1
         $mail.Body[0] | Should -BeExactly $body
     }
+
+    $ItArgs = $PesterArgs.Clone()
+    $ItArgs['Name'] = "Can send mail message from user to self using pipeline " + $ItArgs['Name']
+
+    It @ItArgs {
+        $body = "Greetings from me."
```

Comment:
I would suggest using distinguishable strings in this test i.e. "Greetings from me again" instead of "Greetings from me", "Second test message" instead of "Test message.